### PR TITLE
Quick Fix - map center defaulted to user search center

### PR DESCRIPTION
**Pull request by adisa39**
Quick Fix - map center defaulted to user search center

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -8,4 +8,4 @@ NEXT_PUBLIC_API_URL=http://localhost:8001/api/v1
 NEXT_PUBLIC_PI_SDK_URL=https://sdk.minepi.com/pi-sdk.js
 
 NEXT_PUBLIC_SENTRY_DSN="ADD YOUR SENTRY DSN"
-NEXT_PUBLIC_IMAGE_PLACEHOLDER_URL="ADD YOUR IMAGE PLACEHOLDER URL"
+NEXT_PUBLIC_IMAGE_PLACEHOLDER_URL='/path/to/default/image.png'
```
Comment (swoocn) [inline .env.development:11]: Can you please stop touching this @adisa39..? 😅

Comment (adisa39) [reply .env.development:11]: Noted

File: src/app/[locale]/page.tsx [hunk 2]
```diff
@@ -27,35 +28,66 @@ export default function Index() {
...
-        logger.error('Error getting location on initial load.', { error });
-        setMapCenter(defaultMapCenter);
-        setZoomLevel(2);
+        logger.error('Error fetching user settings data:', { error });
       }
     };
+    getUserSettingsData();
+  }, [currentUser]);
+
+  // useEffect(() => {
```
Comment (fredrick-amoako) [inline src/app/[locale]/page.tsx]: Can we do away with this commented out block? If possible do away with all the commented out codes

Comment (swoocn) [reply src/app/[locale]/page.tsx]: Of course! 👌

File: src/components/shared/map/Map.tsx [hunk 3]
```diff
@@ -72,6 +72,13 @@ const Map = ({
     popupAnchor: [1, -34],
   });
 
+  // Define the crosshair icon for the center of the map
+  const crosshairIcon = new L.Icon({
```
Comment (swoocn) [inline src/components/shared/map/Map.tsx:76]: What is the purpose of this?

File: src/app/[locale]/page.tsx [hunk 4]
```diff
@@ -27,35 +28,66 @@ export default function Index() {
     lat: 0,
     lng: 0,
   });
+  const [searchCenter, setSetSearchCenter] = useState<{ lat: number; lng: number }>({
```
Comment (swoocn) [inline src/app/[locale]/page.tsx:31]: Isn't this [0,0] initialization conflicting with setting the search center coordinates based on the user's IP address? It seems like the search center will always default to the coast of Africa in this case.

**Comment by swoocn** [comment]
This PR is superseded by [https://github.com/map-of-pi/map-of-pi-frontend-react/pull/280](https://github.com/map-of-pi/map-of-pi-frontend-react/pull/280).

**Pull request closed without merging**